ROMFS/px4fmu_common/init.d/rc.sensors

```diff
@@ -111,6 +108,12 @@ then
 	teraranger start -a
 fi
 
+# Blacksheep telemetry
+if param greater TEL_BST_EN 0
```

Comment:
I don't believe this is a sensor of any kind. How about moving it to the top level rcS?

https://github.com/PX4/Firmware/blob/15d1543f95ef0e1929d1fb191f043a7d4bab86d0/ROMFS/px4fmu_common/init.d/rcS#L464